src/PortingAssistantVSExtensionClient2022/Resources/porting-assistant-config.json

```diff
@@ -0,0 +1,20 @@
+{
+  "PortingAssistantConfiguration": {
+    "DataStoreSettings": {
+      "HttpsEndpoint": "https://s3.us-west-2.amazonaws.com/aws.portingassistant.dotnet.datastore/",
```

Comment:
do we want to leave these urls in?

Reply:
the datastore url is public accessible so its fine. We only need to remove logs endpoint data since they are confidential.